src/java.base/share/classes/jdk/internal/classfile/impl/BoundAttribute.java

```diff
@@ -983,4 +985,88 @@ public byte[] codeArray() {
             return classReader.readBytes(payloadStart + 8, codeLength());
         }
     }
+
+    /**
+     * {@return the attribute mapper for a standard attribute}
+     *
+     * @param name the name of the attribute to find
+     */
+    public static AttributeMapper<?> standardAttribute(Utf8Entry name) {
+        // critical bootstrap path, so no lambdas nor method handles here
+        return switch (name.hashCode()) {
```

Comment:
I think we can safely switch over strings, as they are compiled to hashCode switch like what you explicitly have right now. Isn't that the case?

Reply:
Freshly parsed Utf8Entries conversion to String is expensive and unnecessary. We should be very careful when to ask for the conversion as it significantly affects some benchmarks.

Reply:
You are right, I forgot these are Utf8Entry instead of Strings.